### DISCUSSION TITLE
Pull down to load full article

### DISCUSSION THREAD

**Discussion by sozifa** [jocmp/capyreader#325] [Feature Requests]
As the title states - would it be possible to load the full article with a pull down - would make it much easier to use the app single handed
Cheers

**Comment by jocmp** [comment]
good idea. I need to figure out a good way to do swipe gestures via #265 first.

**Reply by sozifa** [reply]
Many thanks for considering

**Comment by jocmp** [comment]
Closing, this feature was added in [2024.09.1051](https://github.com/jocmp/capyreader/releases/tag/2024.09.1051).
